### PR TITLE
Added |2> -> |1> amplitude damping to amplitude damping.

### PR DESCRIPTION
**Pull request by Gabriel-Bottrill**
**Context:**
I recently added `qml.QutritAmplitudeDamping` based on specifications from this [paper](https://doi.org/10.48550/arXiv.1905.10481). This paper only considers the transition from |2> to |0>, in reality on a super conducting device it is more likely to have a |2> to |1> transition. This PR adds a variable and a new Kraus matrix to `qml.QutritAmplitudeDamping` to allow for the simulation of the |2> to |1> transition.

**Description of the Change:**
Adds a new variable, \gamma_3, to `qml.QutritAmplitudeDamping`  that defines the new Kraus matrix K_3. This new Kraus operator allows for the simulation of |2> to |1> relaxations.

**Benefits:**
Allows for simulation of |2> to |1> relaxation. This will make for more accurate simulations and is useful for  measurement error for qutrits.

**Possible Drawbacks:**
This changes `qml.QutritAmplitudeDamping`  and adds an extra parameter, the naming may be a bit more confusing.

**Related GitHub Issues:**
N/A

### CODE REVIEW

**Review by trbromley — APPROVED** [review]
Nice, thanks @Gabriel-Bottrill!

**Review by mudit2812 — COMMENTED** [review]
Just spotted that `default.qutrit.mixed` is missing from the docs for `pennylane/__init__.py:device()` ([see here](https://github.com/PennyLaneAI/pennylane/blob/e1a1ae922942f7404ceb68305a0ccd547359937a/pennylane/__init__.py#L217)). Can you add that in this PR?

**Comment by Gabriel-Bottrill** [comment]
@mudit2812 I added this.

**Comment by Gabriel-Bottrill** [comment]
Hi @albi3ro, @glassnotes, @mudit2812, @trbromley; I didn't tag you all earlier as I didn't intend to make this a full PR, I missclicked when trying to make it a draft. I fixed the issues and it is finalized now, ready for review.

**Review by mudit2812 — APPROVED** [review]

File: pennylane/ops/qutrit/channel.py [hunk 1]
```diff
@@ -246,70 +248,95 @@ class QutritAmplitudeDamping(Channel):
...
     .. note::
 
-        The Kraus operators :math:`\{K_0, K_1, K_2\}` are adapted from [`1 <https://doi.org/10.48550/arXiv.1905.10481>`_] (Eq. 8).
+        When :math:`\gamma_3=0` then Kraus operators :math:`\{K_0, K_1, K_2\}` are adapted from
+        [`1 <https://doi.org/10.48550/arXiv.1905.10481>`_] (Eq. 8).
+
+        The Kraus operator :math:`K_3` represents the :math:`|2 \rangle \rightarrow |1 \rangle` transition which is more
+        likely on some devices [`2 <https://arxiv.org/abs/2003.03307>`_] (Sec II.A).
+
+        To maintain normalization :math:`\gamma_2 + \gamma_3 \leq 1`
```
Comment (glassnotes) [inline pennylane/ops/qutrit/channel.py:269]:
```suggestion
        To maintain normalization :math:`\gamma_2 + \gamma_3 \leq 1`.
```

**Review by glassnotes — APPROVED** [review]
Good catch @Gabriel-Bottrill , thanks for improving this channel implementation!

**Pull request merged**